File: sdkwork-app-sdk-go/types/order_statistics_vo.go
package types

// 订单统计响应
type OrderStatisticsVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	TotalOrders int `json:"totalOrders"`
	PendingPayment int `json:"pendingPayment"`
	PendingShipment int `json:"pendingShipment"`
	PendingReceipt int `json:"pendingReceipt"`
	Completed int `json:"completed"`
	TotalAmount string `json:"totalAmount"`
}

File: sdkwork-app-sdk-csharp/Models/OrderStatisticsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class OrderStatisticsVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? TotalOrders { get; set; }
        public int? PendingPayment { get; set; }
        public int? PendingShipment { get; set; }
        public int? PendingReceipt { get; set; }
        public int? Completed { get; set; }
        public string? TotalAmount { get; set; }
    }
}
